### ISSUE TITLE
R 3.6.3 / 4.x?

### ISSUE THREAD

**Issue by bachlaw** [thequackdaddy/R-OpenBLAS#13]
Microsoft R Open has become unusable (broken package updates, mysterious loss of support, lack of recent R updates) and it is not clear it will be updated beyond 3.5.3.  If it were possible to refresh this effort for newer versions of R it would be a tremendous service to the R Windows community.  Thanks.

**Comment by jamesdalg** [comment]
I also agree. If you could get this working for R 4.0.0, that would be amazing. MRO was great, but it is so backdated that I can't keep everything from breaking.
